bin/resources/GameIndex.yaml

```diff
@@ -56815,9 +56821,11 @@ SLUS-20336:
   region: "NTSC-U"
   compat: 5
 SLUS-20337:
-  name: "X-Men - Wolverine's Revenge"
+  name: "X-Men 2 - Wolverine's Revenge"
```

Comment:
if you're fixing this, the US version is actually called X2 - Wolverine's Revenge